te3/tutorials/data-security/data-security-about.md

```diff
@@ -0,0 +1,386 @@
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 2:</strong> In Tabular Editor, Table Permissions for RLS are visible under the role. New table permissions can be created by right-clicking a Role and selecting <i>'Add Table Partition...'</i></figcaption>
+</figure>
+
+<figure style="padding-top: 15px;">
+  <img class="noscale" src="~/images/data-security/data-security-table-permissions-dax.png" alt="Data Security Create Role" style="width: 550px;"/>
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 3:</strong> Table Permission DAX is visible in the Expression Editor when selecting a Table Permission.</figcaption>
+</figure>
+
+- _<span style="color:#8d7bae">OLS Object Permissions:</span>_ These permissions apply to the primary objects as well as all downstream dependents.
+    - `Read` (Can see / query)
+    - `None` (Cannot see / query)
+    - `Default` (No policy configured; equivalent to `Read`)
+
+<figure style="padding-top: 15px;">
+  <img class="noscale" src="~/images/data-security/data-security-table-permissions-dax.png" alt="Data Security Create Role" style="width: 550px;"/>
+  <figcaption style="font-size: 12px; padding-top: 10px; padding-bottom: 15px; padding-left: 75px; padding-right: 75px; color:#00766e"><strong>Figure 4:</strong> In Tabular Editor, object permissions are accessible in the 'Properties' window, under the 'Translations, Perspectives, Security' heading.</figcaption>
+</figure>
+
+#### 3. __Assign Users to Roles:__ 
+Once configured in the dataset, users must be added to their respective roles.
+- _Power BI:_ Roles are assigned via the __Power BI service__, [in the dataset settings from the Workspace](https://learn.microsoft.com/en-us/power-bi/enterprise-service-admin-rls#manage-security-on-your-model).
+- _SSAS / AAS:_ Roles are assigned via the Role object, by right-clicking and selecting "Edit Members..."
+- _Power BI Embedded:_ You must [generate an embed token](https://learn.microsoft.com/en-us/power-bi/developer/embedded/cloud-rls#generate-an-embed-token).
+
+> [!NOTE]
+> It is [recommended](https://learn.microsoft.com/en-us/power-bi/guidance/powerbi-implementation-planning-security-tenant-level-planning#strategy-for-using-groups) to manage Data Security and Access with [Azure Active Directory Groups](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/how-to-manage-groups). 
+> <br>Using this approach is preferred you can centralize management of security & user segmentation.
+
+#### 4. __Provision Users Access to the Dataset:__ 
+Users must be given dataset access according to the [usage scenario](https://learn.microsoft.com/en-us/power-bi/guidance/powerbi-implementation-planning-usage-scenario-overview).
```

Comment:
I would suggest noting this is Power BI Specific behavior since we have mentioned SSAS and AAS before in the article

Reply:
Agree!